__WIP_pg_schema_review/readme.md

```diff
@@ -0,0 +1,47 @@
+# If you only read one thing, read the `provision.sql` file
```

Comment:
I would make a note that it's generated from .prima file

Reply:
yes I just copied it at the root, I'll actually make a convenience script for this